composer.json

```diff
@@ -19,17 +19,17 @@
         "php": "^7.1|^8.0",
         "ext-pdo": "*",
         "composer/package-versions-deprecated": "^1.8",
-        "doctrine/annotations": "^1.12",
-        "doctrine/cache": "^1.11.0",
+        "doctrine/annotations": "^1.12@dev",
```

Comment:
These need to be reverted to their stable counter parts, I assume they were only present for testing?

Reply:
Correct. If everything else looks good, I'd remove the dev portion to ensure I didn't break anything.